corehq/apps/export/views.py

```diff
@@ -1894,13 +1879,6 @@ class DownloadNewFormExportView(GenericDownloadNewExportMixin, BaseDownloadExpor
     check_for_multimedia = True
     form_or_case = 'form'
 
-    @staticmethod
-    def get_export_schema(domain, export_id):
```

Comment:
This might actually belong with another commit, sorry about that.  Suffice to say that it gets deleted eventually either way.